source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -300,7 +300,7 @@ At least one DOI, ISBN, Online_ISSN, Print_ISSN, Proprietary_ID or URI SHOULD be
 
 .. rubric:: Parent Item Description and Identifiers
 
-When reporting usage on content items like articles and book chapters, it is often desirable to identify the item’s parent item, such as the journal or book it is part of. This next grouping of columns identifies the parents and is used by a small subset of reports.
+When reporting usage on content items like articles and book chapters in an Item Report, it is often desirable to identify the item’s parent item, such as the journal or book it is part of. This next grouping of columns identifies the parents and is used by a small subset of reports. Note that if any parent information is included in an Item Report, the Parent_Data_Type MUST be specified.
```

Comment:
Maybe it should be clarified that this rule applies to each item, e.h. "Note that if any parent information for an item is included in an Item Report, the Parent_Data_Type MUST be specified for the item."?